.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,41 @@
+name: CI/CD pipeline
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        pip install --upgrade pip
+        pip install tox tox-gh-actions setuptools coveralls
```

Comment:
I think would be nice if we considered to pin those versions. Those packages may release major versions with breaking changes that can break Connexion CI/CD pipeline in an unfortunate moment. I know we have more maintainers now, but if we have multiple PRs, we need to coordinate with external contributors to ask them to rebase their PRs or commit to each PR to fix the pipeline. It might be wise to just pin the versions here and avoid the headache. We can still update the versions whenever we need, of course.